modules/apps/blogs/blogs-web/package.json

```diff
@@ -1,6 +1,8 @@
 {
 	"dependencies": {
+		"@liferay/frontend-js-state-web": "*",
 		"frontend-js-web": "*",
+		"item-selector-taglib": "*",
```

Comment:
There is some fake modularity between `blogs-web` and `item-selector-taglib`; they are coupled together via (at least) global events. I made the former depend on the latter because it seems the latter is the more "general" of the two. Note this dependency is solely so that we can get the state atom into `blogs-web`.

Reply:
Yeah... such is the nature of JSPs. Only way we could communicate two different components easily was through global events, so a lot of it has been designed around that concept.